app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java

```diff
@@ -108,12 +152,7 @@ public void createDialog() {
                 .setPositiveButton(R.string.ok, new DialogInterface.OnClickListener() {
                     @Override
                     public void onClick(DialogInterface dialog, int id) {
-                        SharedPreferences.Editor e = p.edit();
-                        e.putBoolean("filter", sw.isChecked());
-                        e.apply();
-                        appSectionsPagerAdapter.notifyFilterChanged();
-                        updateFilteringTextView();
-                        AnalyticsHelper.sendEventHit("Filters", AnalyticsHelper.CLICK, sw.isChecked() ? "Available only" : "All machines");
```

Comment:
I'm not actually looking too carefully, but why do we want to get rid of analytics?